### ISSUE TITLE
TransXNet take such a long time to train

### ISSUE THREAD

**Issue by zhaoyang23333** [LMMMEng/TransXNet#21]
Why does TransXNet take such a long time to train? When I use TransXNet and ConvNeXt respectively as the backbone of Faster R-CNN, under the same learning rate and batch size conditions, the average training time per epoch for ConvNeXt is only 1/5 that of TransXNet.（thank you!）

**Comment by LMMMEng** [comment]
Hi, thank you for your interest. 
I suggest this is a normal phenomenon, as ConvNeXt is a pure ConvNet, while TransXNet is a hybrid model that integrates dynamic convolution and self-attention. Both mixers are slower than the regular depthwise convolution used in ConvNeXt. 
It should be noted that TransXNet is not focused on efficiency, but rather on performance. TransXNet's performance far surpasses that of ConvNeXt on various tasks. 
Moreover, as shown in Tables 10 and 11, our method achieves competitive inference efficiency and memory usage compared to other performance-oriented methods.

**Comment by zhaoyang23333** [comment]
> 您好，感谢您的关注。我认为这是一个正常的现象，因为 ConvNeXt 是一个纯 ConvNet，而 TransXNet 是一个集成了动态卷积和自我注意的混合模型。这两个 mixer 都比 ConvNeXt 中使用的常规深度卷积慢。应该注意的是，TransXNet 并不关注效率，而是关注性能。TransXNet 在各种任务上的性能都远超 ConvNeXt。此外，如表 10 和表 11 所示，与其他面向性能的方法相比，我们的方法实现了有竞争力的推理效率和内存使用。

OK i Got it. Thank you for your reply! Best wishes!
